token-lending/program/src/processor.rs

```diff
@@ -1526,6 +1526,10 @@ fn process_liquidate_obligation(
     Ok(())
 }
 
+const EXECUTE_OPERATION_DATA_SIZE: usize = 9;
+
+const EXECUTE_OPERATION_TAG: u8 = 0u8;
```

Comment:
Maybe name these something more specific to flash loans?

Reply:
Changed them to `FLASH_LOAN_EXECUTE_OPERATION_DATA_SIZE` and `FLASH_LOAN_EXECUTE_OPERATION_TAG`.